docs/conf.py

```diff
@@ -24,7 +24,7 @@
 
 def setup(app):
     app.add_stylesheet(
-        'https://cdnjs.cloudflare.com/ajax/libs/github-fork-ribbon-css/0.2.2/gh-fork-ribbon.min.css'
```

Comment:
I don't think this was supposed to get changed by bumpversion

Reply:
`[bumpversion:file:docs/conf.py]` technically, it is...